### DISCUSSION TITLE
Postal Initialize echoing Database 'postal' already exists

### DISCUSSION THREAD

**Discussion by KalynDavis** [postalserver/postal#2562] [Help with using Postal]
## Describe the bug

When running `postal initialize`, I'm expecting the migrations to run. Instead, I receive this output:
```
Initializing database
Database 'postal' already exists
```

This occurs on the initial installation.

## To Reproduce

1. run `postal initialize`
2. message `Database 'postal' already exists is displayed

## Expected behaviour

Details of the migrations running are expected.

## Environment details 

 - OS: Ubuntu
 - Type: Vultr Docker instance

**Comment by willpower232** [comment]
Have you definitely not run the installation before? Did you accidentally create the database manually when setting up the users?

**Comment by Dave777** [comment]
I have the same problem.

I used this to create and run MariaDB inside Docket container:

```
docker run -d \
--name postal-mariadb \
-p 127.0.0.1:3306:3306 \
--restart always \
-e MARIADB_DATABASE=postal \
-e MARIADB_ROOT_PASSWORD=kjfgjklljbljkgb \
mariadb
```

I guess I need to delete the postal database?

Thinking of using the mysql command : but what would be my user name?
From the above is the user "root" or "postal" ?

`mysql -u postal -p` 

then do I do:

`DROP DATABASE postal;`

Thanks

**Reply by willpower232** [reply]
`-e MARIADB_DATABASE=postal` will pre-create the database so remove that line and you're probably fine also definitely change your root database password now it is in github :sweat_smile:

**Reply by Dave777** [reply]
Thanks for the reply.

So since I have already run the set of commands.

I guess I should run:
`docker container stop  postal-mariadb`

and then :

```
docker run -d \
--name postal-mariadb \
-p 127.0.0.1:3306:3306 \
--restart always \
-e MARIADB_ROOT_PASSWORD=my_strong_password \
mariadb
```

Should that do the job?

**Reply by willpower232** [reply]
You will have to remove the container because `docker run` won't replace it.

As you don't seem to have a volume configured, that should be fine.
